scr/index.html

```diff
@@ -49,6 +49,150 @@ <h1 class="headline-h1 h1Color">I’m Aisha.  <br>
+  <li>Full Stack Dev</li>
+  <li>2015</li>
+</ul>
+</div>
+
+<p class="work-p">Experimental content creation feature that allows users to add to an existing story over the 
+course of a day without spamming their friends.</p>
+<ul class="work-list">
+  <li><a href=""> html</a></li>
+  <li class="desktop_link"><a href=""> Ruby on rails</a></li>
+  <li><a href=""> css</a></li>
+  <li><a href=""> javaScript</a></li>
+</ul>
+<button class="btn">See project</button>
+</div>
+</section>
+</section>
+<!--------------------Works Container3-------------------->
+<section class="work-container1">
+
+  <img src="../Images/3.png" alt="">
+<div class="work">
+<h1>Facebook 360</h1>
+<div class="facebook"> <p>FACEBOOK</p>
+<ul>
+  <li>Full Stack Dev</li>
+  <li>2015</li>
+</ul>
+</div>
+
+<p class="work-p">Exploring the future of media in Facebook's first Virtual Reality app; a place to 
+discover and enjoy 360 photos and videos on Gear VR.</p>
+<ul class="work-list">
+  <li><a href=""> html</a></li>
+  <li class="desktop_link"><a href=""> Ruby on rails</a></li>
+  <li><a href=""> css</a></li>
+  <li><a href=""> javaScript</a></li>
+</ul>
+<button class="btn">See project</button>
+</div>
+</section>
+</section>
+<!--------------------Works Container4-------------------->
+<section class="work-container2" >
+
+  <img src="../Images/4.png" alt="">
+<div class="work">
+<h1>Uber Navigation</h1>
+<div class="facebook"> <p>Uber</p>
+<ul>
+  <li>Lead Developer</li>
+  <li>2018</li>
+</ul>
+</div>
+
+<p class="work-p">A smart assistant to make driving more safe,
+ efficient, and fun by unlocking your most expensive computer: your car.</p>
+ <ul class="work-list">
+  <li><a href=""> html</a></li>
+  <li class="desktop_link"><a href=""> Ruby on rails</a></li>
+  <li><a href=""> css</a></li>
+  <li><a href=""> javaScript</a></li>
+</ul>
+<button class="btn">See project</button>
+</div>
+</section>
+<!--------------------About Me-------------------->
+<section class="about-container" id="About-container">
+<div class="about-r">
+  <h1 class="about-text h1Color">About Myself</h1>
+  <p>Hello I’m a software developer! I can help you build a product , feature or website Look through 
+    some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <p class="about-connect main_color">LET’S CONNECT</p>
+      <div class="about-icon">
+        <ul class="social-media-icons"> 
+          <li><a href="#"><img class="icon" src="../Images/twitter.svg" alt=""></a></li>
+          <li><a href="#"><img class="icon" src="../Images/Linkedin icon.svg" alt=""></a></li>
+          <li><a href="#"><img class="icon" src="../Images/medium.svg" alt=""></a></li>
+          <li><a href="#"><img class="icon" src="../Images/github.svg" alt=""></a></li>
+          <li><a href="#"><img class="icon" src="../Images/angellist.svg" alt=""></a></li>
+        </ul>
+     </div>
+  <button class="btn">See project</button>
+</div>

```

Comment:
You have done a great job with your about myself section! I have a couple of suggestions that could help make it look even closer to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=X3BgW0yrAVqXHazX-0):

- [ ] Please check if you are using the correct `font size` and `color` for your first paragraph. It should be using `color` `#344563` and a `font size` of `16px`. 🎯
- [ ] Instead of using another `p` tag for **"Let's connect"**, consider using a different heading tag such as `h2, h3, etc`. **This can help to improve the overall structure and readability of your content.** 🙌
- [ ] It's also a good idea to check the separation of your social icons and aim to keep them as close as possible to the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=X3BgW0yrAVqXHazX-0). 😃

  ### Your design: 
  ![image](https://user-images.githubusercontent.com/107952724/236304130-e0ca04d5-2326-43d3-9d91-a029dd2bdfb4.png)
  
  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236304230-41bff8a2-84bd-47bb-8c0a-7f98196171e7.png)